src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -2,23 +2,20 @@
 :page-slug: custom-domains
 :page-description: How to configure your Koji to use a custom domain.
 
-The domain name for your Koji is its web address.
-A domain name consists of a *root domain*, such as `+++withkoji.com+++`, and usually one or more subdomains.
-For example, in the domain name `+++withkoji.com/~johnsmith789/my-video-game+++`, `+++~johnsmith789+++` and `+++my-video-game+++` are subdomains.
-The Koji platform assigns a domain name to your Koji when you publish it.
-
-You might want to assign a custom domain name to your Koji.
-A custom domain name can be much easier to remember, and it usually just looks better.
-Custom domains are about where you want to serve the experience.
+When you publish a Koji, it's automatically assigned a permalink based on your user name and lives on the `+++withkoji.com+++` domain.
```

Comment:
The second part of this sentence feels a bit disconnected from the first. Here's one suggestion, but feel free to try variations that might work better.
```suggestion
When you publish a Koji, it's automatically assigned a permalink, based on your user name, that lives on the `+++withkoji.com+++` domain.
```